app/views/provider/debt-management/990175001.html

```diff
@@ -0,0 +1,314 @@
+<!-- Use this page as the index for your project -->
+
+<!-- ADDING CUSTOM CSS - Add your custom CSS or Sass in /app/assets/sass/main.scss -->
+
+<!-- Extends the layout from /views/layout.html -->
+{% extends 'default-layout.html' %}
+
+<!-- Set the page title -->
+{% block pageTitle %}
+Contract details
+{% endblock %}
+
+<!-- Breadcrumbs -->
+{% block beforeContent %}
+{{ breadcrumb({
+items: [
+{
+href: "/provider/landing-page",
+text: "Home"
+},
+{
+href: "/provider/debt-management/debt-contracts",
+text: "Contracts in debt"
+}
+],
+href: "/provider/debt-management/990175001",
+text: "990175001"
+}) }}
+{% endblock %}
+
+<!-- Back link -->
+{% block outerContent %}
+
+{% endblock %}
+
+{% block content %}
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <h1 class="nhsuk-u-margin-bottom-0">
+      Debt Status
+      <span class="nhsuk-caption-xl nhsuk-caption--bottom">Contract Name, 99017500001</span>
+    </h1>
+
+    <hr class="nhsuk-section-break  nhsuk-section-break--l " />
+    <!-- Summary Cards -->
+    <ul class="nhsuk-grid-row nhsuk-card-group">
+      <li class="nhsuk-grid-column-one-third nhsuk-card-group__item">
+        <div class="nhsuk-card nhsuk-no-border">
+          <div class="nhsuk-card__content">
+            <p class="nhsuk-heading-s nhsuk-u-margin-bottom-1">Total debt</p>
+            <p class="nhsuk-body nhsuk-u-margin-bottom-1">£15,000</p>
+            <a href="/provider/debt-management/990175001/debt-details"
+              class="nhsuk-card__link nhsuk-u-font-weight-normal nhsuk-u-font-size-19 nhsuk-link--no-visited-state">Check
+              what you owe</a>
+          </div>
+        </div>
+      </li>
+      <li class="nhsuk-grid-column-one-third nhsuk-card-group__item">
+        <div class="nhsuk-card nhsuk-no-border">
+          <div class="nhsuk-card__content">
+            <p class="nhsuk-heading-s  nhsuk-u-margin-bottom-1">Cut-off date</p>
+            <p class="nhsuk-body nhsuk-u-margin-bottom-1">7 August 2024 </p>
+          </div>
+        </div>
+      </li>
+      <li class="nhsuk-grid-column-one-third nhsuk-card-group__item">
+        <div class="nhsuk-card nhsuk-no-border">
+          <div class="nhsuk-card__content nhsuk-card-custom-action_list">
+            <p class="nhsuk-heading-s nhsuk-u-margin-bottom-1">History</p>
+            <ul class="nhsuk-list">
+              <li>
+                <a href="#"
+                  class="nhsuk-card__link nhsuk-u-font-weight-normal nhsuk-u-font-size-19 nhsuk-link--no-visited-state">View
+                  pay
+                  statements</a>
```

Comment:
Multiple cases where links are wrapping onto different lines.

Reply:
Yes it's because of the default formatter plugin 'prettier' which has ruleset, but may be need to check as per project. what we can use as a common ruleset or tool/plugin.